csp/impl/pandas_perspective.py

```diff
@@ -80,18 +67,27 @@ def _apply_updates(
                 if index_col:
                     row[index_col] = idx
                 if s_has_time_col:
-                    if localize:
+                    if localize or _PERSPECTIVE_3:
                         row[time_col] = pytz.utc.localize(csp.now())
                     else:
                         row[time_col] = csp.now()
+                    if _PERSPECTIVE_3:
+                        row[time_col] = int(row[time_col].timestamp() * 1000)
```

Comment:
The `timestamp() * 1000` logic is scattered throughout all this code. I read its because perspective uses JavaScript and they have a ms precision. Instead of having ~10 places where we're doing this logic `int(dt.timestamp() * 1000)`, can we just define a common helper function called `to_p3_dt` or something? 
We can include it in `perspective_common.py` where we also put the common version parsing logic. Seems less error prone and more readable.

Reply:
Also done.

Reply:
Just FYI - if you switch to Arrow instead of Python Records for inserting, Perspective supports `s`, `ms`, `ns` and `us`.

Reply:
The issue is that it's a pandas table of csp edges, not of raw data. The csp edges are just python types, so I still need the conversion here.